docs/setup/mac.md

```diff
@@ -12,20 +12,18 @@ MetaDescription: Get Visual Studio Code up and running on Mac.
 
 ## Installation
 
-1. [Download Visual Studio Code](https://go.microsoft.com/fwlink/?LinkID=534106) for Mac.
-2. Double-click on the downloaded archive to expand the contents.
-3. Drag `Visual Studio Code.app` to the `Applications` folder, making it available in the `Launchpad`.
-4. Add VS Code to your Dock by right-clicking on the icon and choosing `Options`, `Keep in Dock`.
+1. [Download Visual Studio Code for macOS](https://go.microsoft.com/fwlink/?LinkID=534106) and install it.
```

Comment:
@gregvanl  should i submit the dead-wood removal in another PR. Simply has to go.

Have you tested the install lately? Maybe we could mention those steps are done by default. 

I've never used a mac before and I didn't need to be told how to install a download. Do you really think that's necessary for a programmer?